include/odp/api/spec/traffic_mngr.h

```diff
@@ -443,6 +491,13 @@ typedef struct {
 	 * more of the marking API's. */
 	odp_bool_t marking_colors_needed[ODP_NUM_PACKET_COLORS];
 
+	/** The pkt_prio_mode indicates the packet priority mode, this TM system
+	 * needs to run with. A TM system may or may not support a given mode.
+	 * TM capabilities will indicate support for different modes by the
+	 * platform. Default value of this mode is platform defined.
+	 */
+	odp_tm_pkt_prio_mode_t pkt_prio_mode;
```

Comment:
> Default value of this mode is platform defined.

Specifying a concrete default value would be more useful to an application. I do not see any benefit in a platform defined default here.

Reply:
I think it would be good to specify the default value and make it be the "global" mode (or whatever it ends up getting called). This way odp_tm_requirements_init() will use the default and things are more backward compatible.

Reply:
But what about platforms that cannot support Global mode ? I guess default should be supported.

Reply:
I do not think default needs to be supported. Current implementations will support it, so an unchanged application on top of such implementation would continue to work even after this API change if the default is specified like that. Future ODP implementations may not support it but the application is expected to check the capability field anyway. Also, if the default is not specified, an application cannot rely on the default getting initialized to some sane supported value anyway.

Reply:
Ok. If default need not be supported, its fine with me. I'll make existing mode as default.